Source/WebCore/html/parser/HTMLToken.h

```diff
@@ -122,6 +122,7 @@ class HTMLToken {
     void appendToCharacter(LChar);
     void appendToCharacter(UChar);
     void appendToCharacter(const Vector<LChar, 32>&);
+    template<typename CharacterType> void appendToCharacters(const CharacterType*, unsigned length);
```

Comment:
Why is this named `appendToCharacters`, but the `Vector`-taking one is named `appendToCharacter`? I am pretty sure that “character” here refers to the name of a parser token so it should stay singular.

Can we use `Span` here or does that mess up the optimization?

Reply:
Ok, will make both changes.